test/functional/tests/volumes/test_discard_on_huge_core.py

```diff
@@ -0,0 +1,70 @@
+#
+# Copyright(c) 2020 Intel Corporation
+# SPDX-License-Identifier: BSD-3-Clause-Clear
+#
+
+import pytest
+
+from api.cas import casadm
+from api.cas.cache_config import CacheMode
+from storage_devices.disk import DiskType, DiskTypeSet
+from core.test_run import TestRun
+from test_utils.size import Size, Unit
+
+scsi_dev_size_gb = str(40 * 1024)
+
+
+@pytest.mark.os_dependent
+@pytest.mark.require_plugin("scsi_debug", virtual_gb=scsi_dev_size_gb, opts="1")
+@pytest.mark.parametrizex("cache_mode", CacheMode)
```

Comment:
I think there is no need to do this in more than one, default cache mode. It will make this test too long.